javalib/src/main/scala/java/util/Collections.scala

```diff
@@ -0,0 +1,1131 @@
+package java.util
+
+import java.{lang => jl, io}
```

Comment:
Consider importing `java.io.Serializable` directly, if it's the only thing you use from `java.io`.